packages/oncokb-frontend-commons/src/components/OncoKbCardLevelsOfEvidenceDropdown.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+
+import levelStyles from './level.module.scss';
+import mainStyles from './main.module.scss';
+import collapsibleStyles from './collapsible.module.scss';
+import { Collapse } from 'react-collapse';
+import { levelIconClassNames } from '../util/OncoKbUtils';
+import classnames from 'classnames';
+
+export type LevelsOfEvidenceDropdownProps = {
+    levels: string[];
+    levelDes: { [level: string]: JSX.Element };
+};
+
+export const OncoKbCardLevelsOfEvidenceDropdown: React.FunctionComponent<LevelsOfEvidenceDropdownProps> = (
+    props: LevelsOfEvidenceDropdownProps
+) => {
+    const [levelsCollapsed, updateLevelCollapse] = React.useState(true);
+
+    function handleLevelCollapse() {
+        updateLevelCollapse(!levelsCollapsed);
+    }
```

Comment:
Why do we need a wrapper?

Reply:
My apologies. I think I missed this comment but it is necessary because it is passed into onClick and would produce a type error without it. (Type 'void' is not assignable to type 'MouseEventHandler<HTMLDivElement> | undefined'.)